src/base/Cellar.sol

```diff
@@ -372,41 +361,90 @@ contract Cellar is ERC4626, Ownable, Multicall {
-     * @dev This should be a value out of 1e18 (ie. 1e18 represents 100%, 0 represents 0%).
+     * @notice Emitted when strategists payout address is changed.
+     * @param oldPayoutAddress value strategists payout address was changed from
+     * @param newPayoutAddress value strategists payout address was changed to
      */
-    uint64 public performanceFee = 0.1e18; // 10%
+    event StrategistPayoutAddressChanged(address oldPayoutAddress, address newPayoutAddress);
+
+    /**
+     * @notice Data related to fees.
+     * @param sharePriceHighWatermark Stores the share price to be used as a High Watermark to calculate performance fees.
+     * @param strategistPerformanceCut Determines how much performance fees go to strategist.
+     *                                 This should be a value out of 1e18 (ie. 1e18 represents 100%, 0 represents 0%).
+     * @param strategistPlatformCut Determines how much platform fees go to strategist.
+     *                              This should be a value out of 1e18 (ie. 1e18 represents 100%, 0 represents 0%).
+     * @param platformFee The percentage of total assets accrued as platform fees over a year.
+                          This should be a value out of 1e18 (ie. 1e18 represents 100%, 0 represents 0%).
+     * @param performanceFee The percentage of total assets accrued as platform fees over a year.
+     *                       This should be a value out of 1e18 (ie. 1e18 represents 100%, 0 represents 0%).
+     * @param feesDistributor Cosmos address of module that distributes fees, specified as a hex value.
+     *                        The Gravity contract expects a 32-byte value formatted in a specific way.
+     * @param strategistPayoutAddress Address to send the strategists fee shares.
+     */
+    struct FeeData {
+        uint256 sharePriceHighWatermark;
+        uint64 strategistPerformanceCut;
+        uint64 strategistPlatformCut;
+        uint64 platformFee;
+        uint64 performanceFee;
+        bytes32 feesDistributor;
+        address strategistPayoutAddress;
+    }
 
     /**
-     * @notice Cosmos address of module that distributes fees, specified as a hex value.
-     * @dev The Gravity contract expects a 32-byte value formatted in a specific way.
+     * @notice Stores all fee data for cellar.
      */
-    bytes32 public feesDistributor = hex"000000000000000000000000b813554b423266bbd4c16c32fa383394868c1f55";
+    FeeData public feeData =
+        FeeData({
+            sharePriceHighWatermark: 0,
+            strategistPerformanceCut: 0.75e18,
+            strategistPlatformCut: 0.75e18,
+            strategistPayoutAddress: address(0),
+            platformFee: 0.01e18,
+            performanceFee: 0.1e18,
+            feesDistributor: hex"000000000000000000000000b813554b423266bbd4c16c32fa383394868c1f55"
+        });
+
+    /**
+     * @notice Get the fee data values for this cellar
+     */
+    function getFeeData() external view returns (FeeData memory) {
```

Comment:
Delete this.

A getter function is automatically created for `feeData` by making it `public`.